lms/djangoapps/certificates/models.py

```diff
@@ -854,16 +860,21 @@ class CertificateGenerationCourseSetting(TimeStampedModel):
     for the course.  This is enforced in the UI layer, but
     not in the data layer.
 
+    'language_specific_templates' controls whether certificates for the
+    course should be generated using specific language templates.
```

Comment:
Nit: I would change `should` to `can` since setting this flag does not guarantee that the certificate will be rendered using a language-specific template. I also think it might be clearer if we changed `specific language` to `language-specific`.